getaround_utils/lib/getaround_utils/railties/health_engine.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'rack'
+
+module GetaroundUtils; end
+
+module GetaroundUtils::Railties; end
+
+module GetaroundUtils::Railties::HealthEngine
```

Comment:
🤔 Not sure about the naming...

I would have named it `GetaroundUtils::Railties::Health` and renamed the `app` method into `engine` which makes more sense IMO when calling the helper methods (ie `GetaroundUtils::Railties::Health.release_version`)

❓ WDYT ?